doc/manual.rst

```diff
@@ -3564,10 +3564,25 @@ procedures:
 
 
 Procs as expressions can appear both as nested procs and inside top level
-executable code. The  `sugar <sugar.html>`_ module contains the `=>` macro
+executable code. The `sugar <sugar.html>`_ module contains the `=>` macro
 which enables a more succinct syntax for anonymous procedures resembling
 lambdas as they are in languages like JavaScript, C#, etc.
 
+Lambdas
```

Comment:
The manual does not describe the standard library.

Reply:
But the manual already mentions `sugar.=>` in the context of lambdas, and the new `lambdas.~>` more closely matches the notion of a lambda, so it wouldn't make sense to mention `=>` but not `~>`

Reply:
Hmm true but the manual should only link to "sugar" and "lambdas" without much ado.

Reply:
I reorganized and shortened a bit, grouping both => and ~> under a section `Lambdas`; I don't think it hurts having this in the manual since it's something users coming from other languages (D, js, C# etc) would expect to find, even if we're able to do this in library code as opposed to being built in